src/Rules/PdoStatementExecuteMethodRule.php

```diff
@@ -71,6 +71,16 @@ private function checkErrors(MethodReflection $methodReflection, MethodCall $met
         $args = $methodCall->getArgs();
         if (\count($args) < 1) {
             $parameters = [];
+
+            $calls = $stmtReflection->findPrepareBindCalls($methodCall);
+
+            foreach ($calls as $bindCall) {
+                $args = $bindCall->getArgs();
+                $parameterName = $args[0]->value->value;
+                $parameterTypes = $scope->getType($args[1]->value);
+
+                $parameters[$parameterName] = $queryReflection->resolveParameters($parameterTypes) ?? [];
```

Comment:
we use the same api a few lines below without this `[$parameterName]` offset, which make me feel that there is a bug somewhere

Reply:
`resolveParameters` accepts a Type which is constructed from the array provided into `execute`, since I don't have such an array of values from bind calls, I have to construct it manually (very possible that I did it incorrectly).

If it's possible to construct some kind of Type array that converts bind calls into an array that was passed into the execute call, it would also work.

Reply:
maybe I am mistaken.. came to this conclusion because of the phpstan error in https://github.com/staabm/phpstan-dba/runs/5111061161?check_suite_focus=true of line 98

Reply:
You are right that it currently just puts an empty array there.